menuselect/menuselect_gtk.c

```diff
@@ -52,15 +52,21 @@ static void handle_save(GtkWidget *w, gpointer data);
 static void handle_about(GtkWidget *w, gpointer data);
 static void handle_quit(GtkWidget *w, gpointer data);
 
+static void handle_save_wrapper(void) { handle_save(NULL, NULL); }
+static void handle_about_wrapper(void) { handle_about(NULL, NULL); }
+static void handle_quit_wrapper(void) { handle_quit(NULL, NULL); }
```

Comment:
Why don't we just update the handlers themselves to have the correct signatures instead of creating wrappers?